soft4pes/control/mpc/controllers/rl_grid_mpc_curr_ctr.py

```diff
@@ -0,0 +1,131 @@
+""" Model-predictive controller for RL grid."""
```

Comment:
I would not write _anywhere_ "Model-predictive controller". I would write "Model predictive control" or "MPC" or (maybe in this case) "Finite control set MOC (FCS-MPC)"

Reply:
Solved in next commit